lib/index.js

```diff
@@ -1,22 +1,31 @@
 'use strict'
 
 module.exports = function (crypto) {
```

Comment:
We can modernise the code.

Reply:
Good.
I've also updated the README, and added basic.js example file.